public/js/dashboard/user.js

```diff
@@ -3,7 +3,7 @@ let blockEdit = false;
 function editUser() {
   if (blockEdit) return;
   blockEdit = true;
-  const url = '../api/users/' + searchedUser.id;
+  const url = '/api/users/' + searchedUser.id;
```

Comment:
The URL should not be absolute but relative. In this case it should be `../../api/users/`.